gui/wxpython/gui_core/menu.py

```diff
@@ -305,3 +307,68 @@ def OnItemSelected(self, node):
             label = data['description']
 
         self.showNotification.emit(message=label)
+
+
+class RecentFilesMenu:
+    """Add recent files history menu"""
```

Comment:
Here you should also document the signal, i.e., which event/state change it is signaling and what are the parameters.

Reply:
All right, I added it.